ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -32,7 +36,11 @@ - (NSString *)rac_description {
 @implementation NSString (RACDescription)
 
 - (NSString *)rac_description {
-	return self.description;
+    if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL) {
```

Comment:
I don't think we really need the check for this case, since an `NSString` description is just the string itself.